setup.py

```diff
@@ -19,7 +19,7 @@
 setup(
     zip_safe=True,
     name='cloudify-openstack-plugin',
-    version='2.4.1',
+    version='2.4.1.1',
     author='idanmo',
```

Comment:
Shouldn't the `author` and `author_email` fields be Cloudify-global rather than pointing to a specific person?

Reply:
I like the suggestion, but lets leave it until we make an official decision.